firebase-installations/src/main/java/com/google/firebase/installations/GetAuthTokenListener.java

```diff
@@ -38,12 +38,13 @@ public boolean onStateReached(PersistedFidEntry persistedFidEntry) {
               .build());
       return true;
     }
+    return false;
+  }
 
+  @Override
+  public boolean onException(PersistedFidEntry persistedFidEntry, Exception exception) {
     if (persistedFidEntry.isErrored()) {
```

Comment:
when would it be the case that we'd get an exception without the persistedFidEntry being in the error state?

Reply:
Interrupted Exception 
Exception thrown while persisting. 
Thread failure